types/auth0/index.d.ts

```diff
@@ -1893,6 +1902,10 @@ export class ManagementClient<A = AppMetadata, U = UserMetadata> {
     linkUsers(userId: string, params: LinkAccountsParams): Promise<any>;
     linkUsers(userId: string, params: LinkAccountsParams, cb: (err: Error, data: any) => void): void;
 
+    // User Logs
+    getUserLogs(params: UsersLogsQuery): Promise<Array<LogEvent>>;
+    getUserLogs(params: UsersLogsQuery, cb?: (err: Error, data: Array<LogEvent>) => void): void;
```

Comment:
```suggestion
    getUserLogs(params: UsersLogsQuery, cb: (err: Error, data: Array<LogEvent>) => void): void;
```

Reply:
(when undefined, Promise is used, otherwise callback is required and returns void)